src/main/java/games/strategy/engine/framework/startup/ui/InGameLobbyWatcher.java

```diff
@@ -221,20 +221,20 @@ public void connectionAdded(final INode to) {
             if (portString == null || portString.trim().length() <= 0) {
               portString = "3300";
             }
-            final String message = "Your computer is not reachable from the internet.\r\n"
-                + "Please make sure your Firewall allows incoming connections (hosting) for TripleA.\r\n"
-                + "(The firewall exception must be updated every time a new version of TripleA comes out.)\r\n"
+            final String message = "Your computer is not reachable from the internet.\n"
+                + "Please make sure your Firewall allows incoming connections (hosting) for TripleA.\n"
+                + "(The firewall exception must be updated every time a new version of TripleA comes out.)\n"
                 + "And that your Router is configured to send TCP traffic on port " + portString
                 + " to your local ip address.\r\n"
-                + "See 'How To Host...' in the help menu, at the top of the lobby screen.\r\n"
-                + "The server tried to connect to your external ip: " + addressUsed + "\r\n";
+                + "See 'How To Host...' in the help menu, at the top of the lobby screen.\n"
+                + "The server tried to connect to your external ip: " + addressUsed;
```

Comment:
The changes in this file appear to be independent of the feature you are adding.  Since this could be a potentially long-running branch, any reason not to just submit them in a separate PR?  They seem useful on their own.

Reply:
Will do